Eatery/AppDelegate.swift

```diff
@@ -13,7 +13,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     var eateryTabBarController: EateryTabBarController!
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions:  [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
-
+        
```

Comment:
Remove space

To save you the effort of having to remove it everywhere, enable "automatically trim trailing whitespace" and "including whitespace only lines": 
<img width="912" alt="Screen Shot 2019-11-02 at 13 06 24" src="https://user-images.githubusercontent.com/1882991/68074463-a43b6b00-fd71-11e9-903f-356913af98ee.png">

Reply:
is swiftlint not setup on repo?